src/routes/auth.js

```diff
@@ -0,0 +1,90 @@
+/** @type function(Services): Promise<void> */
+export const auth = async ({ application: { server, auth } }) => {
```

Comment:
`application` used as a glue for everything. It is an obvious fat interfaced object. Avoid using such a thing. It is possible to have a DI with only `auth` here. But IMO it is better to have instance resolved by string or DI by token, to not depend on AuthService interface directly